core/templates/services/question-validation.service.ts

```diff
@@ -16,22 +16,31 @@
  * @fileoverview Service to validate a question.
  *
  */
+import { Injectable } from '@angular/core';
+import { downgradeInjectable } from '@angular/upgrade/static';
 
-require(
-  'components/state-editor/state-editor-properties-services/' +
-  'state-editor.service.ts');
+import { StateEditorService } from
+  // eslint-disable-next-line max-len
+  'components/state-editor/state-editor-properties-services/state-editor.service';
```

Comment:
Imports don't count towards max-len violations, you can keep them in one line
```suggestion
import { StateEditorService } from 'components/state-editor/state-editor-properties-services/state-editor.service';
```

Reply:
done!